azext_iot/central/commands_enrollment_group.py

```diff
@@ -0,0 +1,300 @@
+# coding=utf-8
+# --------------------------------------------------------------------------------------------
+# Copyright (c) Microsoft Corporation. All rights reserved.
+# Licensed under the MIT License. See License.txt in the project root for license information.
+# --------------------------------------------------------------------------------------------
+# Dev note - think of this as a controller
+
+from typing import List
+from azext_iot.constants import CENTRAL_ENDPOINT
+from azext_iot.central.providers import CentralEnrollmentGroupProvider
+from azext_iot.common import utility
+from azext_iot.central.common import API_VERSION
+from azext_iot.central.models.ga_2022_07_31 import EnrollmentGroupGa
+
+
+def get_enrollment_group(
+    cmd,
+    app_id: str,
+    group_id: str,
+    entry: str = None,
```

Comment:
tbh the typing should be Optional[str]

Reply:
They should behave the same in this scenario? It's like `entry: str | None = None`? But I can fix this.